Maquetacion/index.html

```diff
@@ -57,8 +57,8 @@ <h2 class="text title-card">Polymer</h2>
       </div>
     </div>
     <h1 class="title">Back-End Tools</h1>
-    <div class="front">
-      <div class="tech">
+    <div class="container">
+      <div class="tech item1">
```

Comment:
You can remove the `item1` class name if you're not using it anywhere. 👍

Reply:
👍